jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/DatePicker.java

```diff
@@ -42,9 +42,9 @@ public class DatePicker extends UIBaseElement<DatePickerAssert> implements ISetu
     private static final String MONTH_LIST_WITHOUT_EXPANDER =
             "//div[@class='v-date-picker-table v-date-picker-table--month theme--light']/table";
     private static final String CANCEL = "//div[@class='v-picker v-card v-picker--date theme--light']" +
-            "//span[text()[contains(.,'Cancel')]]";
```

Comment:
Ok and Cancel buttons are not a part of DatePickers, they should be moved to custom class in vuetify-tests project

Reply:
Buttons are still a part of a DatePicker class - according to this request they should not be.
You need to remove all buttons related activities from DatePicker and DatePickerMonth classes (eg getOkButton, getCancelButton, okButton, cancelButton